build.cake

```diff
@@ -129,11 +130,25 @@ Task("Build")
 Task("Pack")
     .Does(() =>
 {
-    var packDestDir = "src/bin";
-    PaketPack(packDestDir, new PaketPackSettings {
-        Version = isReleaseBranch ? gitVersion.MajorMinorPatch : gitVersion.NuGetVersion,
-        BuildConfig = configuration
-        });
+    var msBuildSettings = new MSBuildSettings {
+        Verbosity = Verbosity.Normal,
+        ToolVersion = MSBuildToolVersion.VS2017,
+        Configuration = configuration,
+        PlatformTarget = PlatformTarget.MSIL
+    };
+    var project = "./src/ControlzEx/ControlzEx.csproj";
+
+    MSBuild(project, msBuildSettings
+      .WithTarget("pack")
+      .WithProperty("PackageOutputPath", "bin")
```

Comment:
This previously used `var packDestDir = "src/bin"`, now it seems to use `bin` relative to the project being built.
That should also explain the missing nupkg on appveyor.

Reply:
Fixed path, should now be okay